WooCommerce/Classes/Analytics/WooAnalytics.swift

```diff
@@ -243,6 +243,7 @@ private extension WooAnalytics {
         updatedProperties[PropertyKeys.blogIDKey] = site?.siteID
         updatedProperties[PropertyKeys.wpcomStoreKey] = site?.isWordPressComStore
         updatedProperties[PropertyKeys.ecommerceTrialKey] = site?.wasEcommerceTrial
+        updatedProperties[PropertyKeys.planKey] = site?.plan
```

Comment:
nit: Kindly consider adding unit tests for the new `was_ecommerce_trial` and `plan` properties.

Reply:
I looked into adding tests when adding `was_ecommerce_trial`, but looks like we are using the site from `ServiceLocator.stores.sessionManager`. I don't think we should inject the store manager to `WooAnalytics`, because that would cause circular dependency. Let me know if you have a suggestion to handle this case.

Reply:
Oh, thank you for the explanation.

>  Let me know if you have a suggestion to handle this case.

We could extract the properties handling methods into a separate use case and unit test it. But I guess this is not a priority now as we don't alter this code often.